### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by Dolnay-Adam**

### CODE REVIEW

**Comment by Dolnay-Adam** [comment]
Requesting review

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (ardangor) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:35]: Very bad job bro

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -14,6 +14,8 @@ public class TorpedoStore {
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```
Comment (ardangor) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:17]: Pathetic attempt, its not even secure bro

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 3]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
...
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (ardangor) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:45]: lol, just look at the code in the mirror bro

**Review by ardangor — CHANGES_REQUESTED** [review]
I dont even care bro

**Review by ardangor — APPROVED** [review]
I approve!

**Pull request merged**